docs/usage/configuration.md

```diff
@@ -4,6 +4,42 @@
 This page is current under construction!
 ```
 
+## Providing Your Own Configuration Settings
+
+While PythonTA comes with its own default configuration settings, you can provide either a `dict` or the file name of the config file containing the configuration options you want to override.
+
+When providing your own configuration settings (either a `dict` or file), you just need to provide a minimal configuration file that contains only the configuration options you wish to override. PythonTA will use its default options for all other options.
```

Comment:
Adding this text is great!! My only comment is it would be good to specify that this file is in the toml file format (and provide a link to the TOML specification), since that may be unfamiliar for users.